oci-library/oci-hol/java-management/setup-a-fleet/setup-a-fleet.md

```diff
@@ -53,15 +53,11 @@ In this lab, you will:
 
   ![image of create fleet confirm creation](/../images/create-fleet-create.png)
 
-9. There are **2 files** to be downloaded. Click **Download Install Key** to download the install key (response file) and click the link to **Download the management agent software**.
+9. Click **Download Install Key** to download the install key. The management agent software will be downloaded in the next lab.
```

Comment:
I am thinking of rephrasing this to the following:

"Click **Download Install Key** to download a response file containing the agent install key. This response file will be used in the next lab for management agent installation"

See if this statement makes sense

Reply:
Hijacking this thread for line 46 also:
 _"In the **Maximum Installations** field, specify a number that indicates the maximum number of installations that can be associated with the key. The default value is 1000, for this example enter **10**."_
 
 Can you remove the mention of default value of 1000? I think this default value will change in the next sprint, pls help me to update too. thx

Reply:
> I am thinking of rephrasing this to the following:
> 
> "Click **Download Install Key** to download a response file containing the agent install key. This response file will be used in the next lab for management agent installation"
> 
> See if this statement makes sense

Yup, makes sense! Should we perhaps standardise the use of "response file" and "install key" for this Lab 3 and the next Lab 4? In this case I'm thinking "response file" would be clearer, but OCI console uses "install key" so I was also concerned that using "response file" might be confusing for the user

Reply:
Yeah that is the confusing/complicated part unfortunately. Using either one on its own, without context of the other is confusing :/

Install key: An install key is issued against your identity domain and validates the authenticity of the installation.
Response file: The Management Agent installation script uses a response file to read the agent parameters specific to your environment.

Slightly more detailed info:
https://docs.oracle.com/en-us/iaas/management-agents/doc/install-management-agent-chapter.html#GUID-92777625-6549-4D8E-A27D-C1C5583071CA 

Probably a short paragraph explaining how install key and response file works? then from there onwards use the term "response file" instead. Any better suggestion?

Reply:
Yes, sounds good to clarify the difference between install key and response file. I'll add in a paragraph to clarify, and then refer to them accordingly!

Reply:
My thoughts are that it might be better to explain install key/response file and how they work with the mgmt agent software in Lab 4 and not in Lab 3, if not might need to repeat in both lab 3 and 4 which I think is not necessary. In Lab 3, the user is setting up the fleet and downloading the key, which are simple steps (easing them into it). Only in Lab 4 during the rpm installation would it be more pertinent to explain to user what the key/response file is doing + all the definition/explanations.

Reply:
yup agree that lab 4 is a better place